### PR TITLE
Add neanias/everforest-nvim

### PR DESCRIPTION
**Pull request by adam-devel**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2374,6 +2374,13 @@
             "summary": "A tree-sitter grammar for nu-lang, the language of nushell",
             "license": "MIT",
             "rockspec_template": "nurr/resources/tree-sitter-nu.template"
+        },
+        {
+            "name": "neanias/everforest-nvim",
+            "shorthand": "everforest-nvim",
+            "dependencies": [],
+            "summary": "A Lua port of the Everforest colour scheme",
+            "license": "No License"
```
Comment (mrcjkb) [inline plugins.json:2383]: note: I would suggest requesting that the plugin author add a license.
According to the [GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license):

> You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work.

**Review by mrcjkb — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 1 above)
Comment (adam-devel) [reply plugins.json:2383]: Done !

Comment (mrcjkb) [inline plugins.json:2383]:
```suggestion
            "license": "MIT"
```

**Review by mrcjkb — CHANGES_REQUESTED** [review]
needs a rebase.

**Review by mrcjkb — APPROVED** [review]

**Pull request merged**